tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/arena/PtrUtil.scala

```diff
@@ -11,6 +11,12 @@ import at.forsyte.apalache.tla.types.tla
  */
 object PtrUtil {
 
+  def getCellMap(ptrs: Seq[ElemPtr]): Map[ArenaCell, Seq[ElemPtr]] =
```

Comment:
Seems like this is always called as

```scala
PtrUtil
      .getCellMap(flatPtrs)
      .toSeq
      .map { case (cell, ptrs) =>
        PtrUtil.mergePtrs(cell, ptrs)
      }
```

Why not extract this larger piece of code?

Reply:
I thought about that as well, and we certainly can, but I'm not sure if it's always going to be used like that.

Reply:
up to you :)